templates/swagger/v1_json.tmpl

```diff
@@ -15622,6 +15622,12 @@
         "password"
       ],
       "properties": {
+        "created_at": {
+          "description": "For explicitly setting the user creation timestamp. Useful when users are\nmigrated from other systems. When omitted, the user's creation timestamp\nwill be set to \"now\".",
```

Comment:
`make swagger` even copies the newlines?
Perhaps we shouldn't use newlines in that comment then…
Or we start a new line once a sentence ends.